src/DataBaseTree/ViewModels/MetadataTree/TreeViewModel.cs

```diff
@@ -7,17 +7,29 @@
 using DBManager.Application.Utils;
 using DBManager.Application.ViewModels.General;
 using DBManager.Application.ViewModels.MetadataTree.TreeItems;
+using DBManager.Application.ViewModels.Windows;
+using Ninject;
 
 namespace DBManager.Application.ViewModels.MetadataTree
 {
     public class TreeViewModel : ViewModelBase
     {
         private ICommand _setSelectedItemCommand;
         private ICommand _copyItem;
+        private ICommand _showDataGenerationConfigWindow;
 
         private TreeViewItemViewModelBase _selectedItem;
 
         public ObservableCollection<TreeViewItemViewModelBase> RootItems { get; }
+        public ICommand ShowDataGenerationConfigWindow => _showDataGenerationConfigWindow ?? (_showDataGenerationConfigWindow = new RelayCommand(s => ExecuteShowDataGenerationConfigWindow()));
```

Comment:
move to other commands please (ordering) and blank lines before and after member